### DISCUSSION TITLE
Support for SolidJS

### DISCUSSION THREAD

**Discussion by thedanchez** [glideapps/glide-data-grid#789] [Ideas]
Just wanted to plant this seed as food for thought and to gauge interest. I feel like it would be cool to have a port of Glide Data Grid for SolidJS considering how close it resembles React. It's also a much more performant piece of technology by comparison across many metrics.

I may have time down the road to maybe try doing it myself (can't make any promises on my free time though), but was curious if it was something that was on the radar? Or would something like that ever be considered for future possibilities?

**Comment by jassmith** [comment]
Would love to do this. As you noted, time is the issue.

**Reply by thedanchez** [reply]
@jassmith just curious, how big of a lift do you think the porting effort would be?

**Reply by jassmith** [reply]
eh, port is probably small, but that is a hard fork. Common core is a bigger lift but probably much better long term as it wont be a total fork.

**Comment by Borderliner** [comment]
I'd love to see this on SolidJs!
